lib/command.h

```diff
@@ -147,6 +147,10 @@ enum node_type {
 	FORWARDING_NODE,	 /* IP forwarding node. */
 	PROTOCOL_NODE,		 /* protocol filtering node */
 	MPLS_NODE,		 /* MPLS config node */
+	SR_NODE,		 /* SR node */
```

Comment:
same

Reply:
This is also same topic about `segment-routing` cli.